test/03_before_build/setup.py

```diff
@@ -1,25 +1,6 @@
 from setuptools import setup, Extension
 import sys, os
 
-if sys.argv[-1] != '--name':
```

Comment:
We'd still like to keep the tests, here, actually. We just don't need to guard it with the `if sys.argv[-1] != '--name':`. So please keep the body of the if-statement without the condition (but dedented, to be valid Python)